packages/common/hangul/src/utils.en.md

```diff
@@ -0,0 +1,40 @@
+---
+title: utils
+---
+
+# hasBatchim
+
+Check if a hangul string has a [final consonants (batchim)](https://en.wikipedia.org/wiki/Hangul_consonant_and_vowel_tables) at the end.
```

Comment:
`hangul/src/utils.ts` is not exported from `@toss/hangul` so I think it doesn't have to be added to our documentation.